### ISSUE TITLE
Add Bourbon

### ISSUE THREAD

**Issue by LizJohnston** [TrackYourself/track-yourself#22]
Add node-bourbon to project, including dependencies. I may need help with this.

**Comment by joshcanhelp** [comment]
I'm around tomorrow morning and can be after class as well ... or Friday.

**Comment by LizJohnston** [comment]
Yes, please. I'm nervous, even though I most likely can do it without messing anything up.

**Comment by joshcanhelp** [comment]
I'll be in around 10 @LizJohnston, happy to help guide you through the process then

**Comment by joshcanhelp** [comment]
Ready when you are @LizJohnston

**Comment by LizJohnston** [comment]
I added bourbon via "npm install node-bourbon --save-dev", but when I test it with grunt, I'm getting this error: track-yourself/app/sass/main.sass:1: error: file to import not found or unreadable: "bourbon." The sass is not installing to css. It could be a problem with the way I installed it or it may be that this particular version of angular and/or node needs me to use the SCSS syntax instead. I think node uses libsass, which just very recently starting allowing the white-space syntex of Sass instead of only accepting the SCSS syntex.
